src/main/java/com/mongodb/hibernate/dialect/MongoDialect.java

```diff
@@ -21,7 +21,7 @@
 import org.hibernate.engine.jdbc.dialect.spi.DialectResolutionInfo;
 
 /**
- * A MongoDB {@link Dialect} for {@linkplain #getMinimumSupportedVersion() version 6.0 and above}.
+ * A MongoDB {@link Dialect} for version 6.0 and above.
```

Comment:
You added `{@linkplain #getMinimumSupportedVersion() version 6.0 and above}`, but then removed it. I did not find any explanation behind the change, nor can I think of one on my own. why was the link to `getMinimumSupportedVersion` removed?

Reply:
I found it a little bit weird. If it is a pointer to some constant, it is a good idea, but I don't think it is a good idea to link to a method, right? Also, I think using `version 6.0 and above` aligns with existing Javadoc pattern for other dialects.

Reply:
I reverted it back to original state